### PR TITLE
Add german translation guidelines

### PR DESCRIPTION
**Pull request by kolaente**
Because I kept seeing the same common questions over and over again (as a proofreader for the german translation) I figured I might as well document the answer properly to have something to point people to. For way too long, this has been [a gist](https://gist.github.com/kolaente/0680c0e6bfa96a1d2183ccb32fff0b5c) which is not really ideal because not everyone has access to it. Also this is mostly what I think it should be, most of it is distilled from discussions and my own work of translating gitea.

I didn't really know where to put it so I just put it under "development" - maybe it would make sense to add a "translation" section? I think this could be useful for other languages as well. And we don't have any documentation about the translation at all at the moment.

---

Pinging @jonasfranz @6543 @silverwind for review

Das hier sind im wesentlichen nur Notizen von mir in einen docs-Artikel kopiert, guckt daher bitte nochmal drüber um das in eine vernünftige Form zu bringen. Ein paar Dinge davon sollten wir vielleicht vorher noch Diskutieren (und deshalb ist dieser Satz hier auch auf deutsch).

### CODE REVIEW

File: docs/content/doc/developers/translation-guidelines-german.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Allgemeines
+
+Anrede: Wenig förmlich:
+* "Du"-Form
+* Keine "Amtsdeusch"-Umschreibungen, einfach so als ob man den Nutzer direkt persönlich ansprechen würde
+
+Genauer definiert:
+* "falsch" anstatt "nicht korrekt/inkorrekt"
+* Benutzerkonto oder Konto? Oder Account?
```
Comment (kolaente) [inline docs/content/doc/developers/translation-guidelines-german.md]: Dazu gab es irgendwann vor viel zu langer Zeit eine Diskussion im Discord, ich glaube das hatten wir nie so richtig geklärt? Auch wenn ich das jetzt nur ungern wieder aufkochen will, sollten wir das vielleicht mal so langsam entscheiden.

**Comment by kolaente** [comment]
Ein bisschen Diskussion gab es auch schon in crowdin: https://crowdin.com/project/gitea/discussions/4

**Comment by jolheiser** [comment]
I think this is probably good information to have, I'm just not sure it makes sense to put in the developer section of the English documentation.

What about a link to the gist in the [localization page](https://docs.gitea.io/en-us/localization/), or maybe a German page and then linking _that_ in the localization page?

**Comment by kolaente** [comment]
@jolheiser I think a dedicated page in this repo would be better suited than the gist since everyone could do a PR to update it but that's not possible with a gist.

I didn't see the localization page, only looked for a page named "Translation" which ofc does not exist :slightly_smiling_face: 

I still think it should be visible in the menu somehow since that would make it far more discoverable when just skimming manually through the docs. What do you think about creating a new section "Localization" and moving the existing localization page and this new page to it?

**Comment by 6543** [comment]
why not add a translation page into docs witch each lang can have it's own "translation" with additional stuff ... ?
-----
Warum nicht eig. eine eigene Seite/Topic "Translation"/"Übersetzen" in die docs einfügen. Dort könnte man spezifisch für jede sprache was machen - so wie die info dieses pulls ?

**Review by khmarbaise — APPROVED** [review]

**Review by techknowlogick — DISMISSED** [review]
Blocking per @6543 as I agree we should have a separate section in docs for translations as we can then add more pages for other languages

File: docs/content/doc/developers/translation-guidelines-german.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+* Pull-Request
+* Time-Tracker
+* E-Mail-Adresse (siehe Duden)
+
+## Artikeldefinitionen für Anglizismen
+
+* _Der_ Commit (m.)
+* _Der_ Branch (m.)
+* _Der_ Issue (m.)
```
Comment (delvh) [inline docs/content/doc/developers/translation-guidelines-german.md]: Ich würde eher sagen, rein intuitiv ist es _das_ Issue. Der Issue hört sich meiner Meinung nach an wie "der Gerät".

File: docs/content/doc/developers/translation-guidelines-german.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* _Der_ Fork (m.)
+* _Das_ Repository (n.)
+* _Der_ Pull-Request (m.)
+* _Das_ Token (n.)
+* _Das_ Review (n.)
+
```
Comment (delvh) [inline docs/content/doc/developers/translation-guidelines-german.md]: Vielleicht sollten noch die folgenden Anglizismen geadded werden:
- _Der_ Merge (m.)
- _Der_ Key (m.)
- _Der_ Squash (m.)
- _Der_ Committer (m.)

File: docs/content/doc/developers/translation-guidelines-german.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## Formulierungen in Modals und Buttons
+
+Es sollten die gleichen Formulierungen auf Buttons und Modals verwendet werden.
+
+Beispiel: Wenn der Button mit `löschen` beschriftet ist, sollte im Modal die Frage lauten `Willst du das wirklich löschen?` und nicht `Willst du das wirklich entfernen?`. Gleiches gilt für Success/Errormeldungen nach der Aktion.
+
+## Trennungen
+
+* Pull-Request
```
Comment (delvh) [inline docs/content/doc/developers/translation-guidelines-german.md]: Ich weiß nicht, aber meiner Meinung nach gehört da kein "-" zwischen Pull-Request (**Pull Request**).
Bei Time-Tracker und E-Mail-Adresse stimme ich hingegen zu.

**Review by delvh — COMMENTED** [review]
Hopefully it'll get merged someday

File: docs/content/doc/developers/translation-guidelines-german.md (hunk 3 above)
Comment (6543) [reply docs/content/doc/developers/translation-guidelines-german.md]: Der Token oder?

File: docs/content/doc/translation/guidelines.de-de.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+* Pull-Request
+* Time-Tracker
+* E-Mail-Adresse (siehe Duden)
+
+## Artikeldefinitionen für Anglizismen
+
+* _Der_ Commit (m.)
+* _Der_ Branch (m.)
+* _Der_ Issue (m.)
```
Comment (6543) [inline docs/content/doc/translation/guidelines.de-de.md]:
```suggestion
* _Das_ Issue (n.)
```

File: docs/content/doc/translation/guidelines.de-de.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+* _Der_ Fork (m.)
+* _Das_ Repository (n.)
+* _Der_ Pull-Request (m.)
+* _Das_ Token (n.)
```
Comment (6543) [inline docs/content/doc/translation/guidelines.de-de.md]:
```suggestion
* _Der_ Token (m.)
```

File: docs/content/doc/translation/guidelines.de-de.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+* _Das_ Review (n.)
```
Comment (6543) [inline docs/content/doc/translation/guidelines.de-de.md:52]:
```suggestion
* _Das_ Review (n.)
* _Der_ Key (m.)
* _Der_ Committer (m.)
```

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@kolaente hab die Vorschläge angewandt ...

**Comment by 6543** [comment]
> why not add a translation page into docs witch each lang can have it's own "translation" with additional stuff ... ?

# solved

File: docs/content/doc/translation/guidelines.de-de.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
+---
+date: "2021-01-22T00:00:00+02:00"
+title: "Übersetzungs Richtlinien"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "translation"
+    name: "Übersetzungs Richtlinien"
```
Comment (kolaente) [inline docs/content/doc/translation/guidelines.de-de.md]:
```suggestion
    name: "Übersetzungsrichtlinien"
```

File: docs/content/doc/translation/guidelines.en-us.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+draft: false
+menu:
+  sidebar:
+    parent: "translation"
+    name: "Translation Guidelines"
+    weight: 70
+    identifier: "translation-guidelines"
+---
+
+It is used to provide a common set of rules to make sure the translation is consistent.
```
Comment (kolaente) [inline docs/content/doc/translation/guidelines.en-us.md]:
```suggestion
This place is used to provide a common set of rules to make sure the translation is consistent.
```

I'd list all available translation guidelines (well, German only atm) here so people don't need to switch their language to find it.

**Review by kolaente — COMMENTED** [review]
Ping @6543

File: docs/content/doc/translation/guidelines.en-us.md (hunk 9 above)
Comment (6543) [reply docs/content/doc/translation/guidelines.en-us.md]: we could list langauges right ...

Comment (6543) [inline docs/content/doc/translation/guidelines.en-us.md]:
```suggestion
It is used to provide a common set of rules to make sure the translation is consistent.

* [German](https://docs.gitea.io/de-de/übersetzungs-richtlinien/)
```

File: docs/content/doc/translation/guidelines.en-us.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+  sidebar:
+    parent: "translation"
+    name: "Translation Guidelines"
+    weight: 70
+    identifier: "translation-guidelines"
+---
+
+This place is used to provide a common set of rules to make sure the translation is consistent.
+
+* [German](https://docs.gitea.io/de-de/übersetzungs-richtlinien/)
```
Comment (6543) [inline docs/content/doc/translation/guidelines.en-us.md]:
```suggestion
* [German](/de-de/übersetzungs-richtlinien/)

```

**Review by kolaente — COMMENTED** [review]
LGTM

**Pull request merged**